Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,27 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            int temp;
+            // traverse = to array length
+            for (int i = 0; i < array.Length - 1; i++)
+            {
+                // traverse i + 1 to array length
```

Comment:
We don't need redundant comments, code speaks for itself

Reply:
One of your conditions is incorrect inside of the InsertAt method.

you have 
if (array.Length == 0 && index > 0 || **index > array.Length - 1** || index < 0) ...

but this should be
if (array.Length == 0 && index > 0 || **index < array.Length - 1** || index < 0) ...